src/Microsoft.CodeAnalysis.Analyzers/UnitTests/MetaAnalyzers/CompareSymbolsCorrectlyTests.cs

```diff
@@ -1248,9 +1248,156 @@ public int GetHashCode(object o1, object o2)
             }.RunAsync();
         }
 
+        [Fact]
+        public async Task RS1024_CustomComparer()
```

Comment:
I don't know when this was done but it seems that all tests were renamed to follow the `Async` suffix pattern, could you please update your tests?

Reply:
That looks weird. We shouldn't follow `Async` suffix for tests.

Reply:
I agree, don't know why it was done.